# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/EarlyHintPreloader.h
# Commit: 7cdf3cef2773
# Full Hash: 7cdf3cef27736a14fd8f9db58522da37e103d984
# Author: Manuel Bucher <manuel@mozilla.com>
# Date: 2022-12-02 21:26:33
# Regressor Bug: 1771867
# File Overlap Count: 1
# Description:
#   Bug 1771867 - Early Hints Phase 2 - Part 3: Add mechanism to notify EarlyHintPreloader that HttpChannelParent is ready to complete the redirect r=necko-reviewers,kershaw,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D161178
# ==============================================================================

diff -r 5090eae24a5c -r 7cdf3cef2773 netwerk/protocol/http/EarlyHintPreloader.h
--- a/netwerk/protocol/http/EarlyHintPreloader.h	Fri Dec 02 09:45:24 2022 +0000
+++ b/netwerk/protocol/http/EarlyHintPreloader.h	Fri Dec 02 09:45:24 2022 +0000
@@ -67,6 +67,8 @@
   // after all, because the final response returns a non-2xx status code.
   nsresult CancelChannel(nsresult aStatus);
 
+  void OnParentReady(nsIParentChannel* aParent, uint64_t aChannelId);
+
  private:
   explicit EarlyHintPreloader(nsIURI* aURI);
   ~EarlyHintPreloader() = default;